src/Tooltip/Arrow.tsx

```diff
@@ -0,0 +1,93 @@
+import { Side } from "@floating-ui/react-dom-interactions";
+import clsx from "clsx";
+import React, { forwardRef } from "react";
+
+import { makeStyles, useTheme } from "../theme";
+import { getBackgroundColor, getBorderColor, StyleProps } from "./styles";
+import { TooltipProps } from "./Tooltip";
+
+const useStyles = makeStyles<StyleProps>((theme) => ({
```

Comment:
I think it can be moved to `styles.ts` as `useArrowStyles`

Reply:
👍🏻 fixed that